core/iwasm/compilation/aot_emit_function.c

```diff
@@ -964,7 +965,9 @@ aot_compile_op_call(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
 
         /* Set calling convention for the call with the func's calling
            convention */
-        LLVMSetInstructionCallConv(value_ret, LLVMGetFunctionCallConv(func));
+        if (LLVMGetValueKind(func) == LLVMFunctionValueKind)
```

Comment:
When will the `LLVMGetValueKind(func)` be not equals to `LLVMFunctionValueKind`?

Reply:
L935. `if (!(func = LLVMBuildBitCast(comp_ctx->builder, func_ptr, ..., `.